audinterface/core/utils.py

```diff
@@ -391,6 +393,23 @@ def to_array(value: typing.Any) -> np.ndarray:
     return value
 
 
+def to_samples(
+        duration: pd.Timedelta,
+        sampling_rate: int,
+) -> int:
+    r"""Convert duration to samples.
+
+    Args:
+        duration: duration in seconds
+        sampling_rate: sampling rate in Hz
+
+    Returns:
+        duration in samples
+
+    """
+    return int(round(duration.total_seconds() * sampling_rate))
```

Comment:
In our case round() should always return an integer, so we do not necessarily need the cast.

round: "The return value is an integer if ndigits is omitted or None."

Reply:
This line is no longer part of the updated code as we use `audmath.samples()` and do no longer require `to_samples()`.